google/cloud/forseti/services/dao.py

```diff
@@ -236,7 +236,7 @@ class Resource(base):
         policy_update_counter = Column(Integer, default=0)
         display_name = Column(String(256), default='')
         email = Column(String(256), default='')
-        data = Column(Text)
+        data = Column(Text(16777215))
```

Comment:
I recommend making changes like this in a different PR, so we can track them separate from the installer changes.

Reply:
Removed, will create another PR to fix this.